api/nodes/serializers.py

```diff
@@ -110,6 +118,7 @@ def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
     def create(self, validated_data):
+        validated_data.pop('type')
```

Comment:
@sloria, regarding `def validate_id:`
accessing self._args[0] returns the Node, User, etc. that you're trying to update.  I was comparing self._args[0]._id to the `value` supplied in the field.

You can get the 'kwargs' with self.context['request'].parser_context['kwargs'] however the name will vary, i.e. 'node_id' or 'user_id' making it harder to use this if we're validating `id` in the base serializer.